honeycomb/core/src/main/java/org/commonjava/o11yphant/honeycomb/DefaultHoneycombManager.java

```diff
@@ -69,6 +72,17 @@ public DefaultHoneycombManager( HoneycombConfiguration honeycombConfiguration,
         }
     }
 
+    public DefaultHoneycombManager( HoneycombConfiguration honeycombConfiguration,
+                                    DefaultTraceSampler defaultTraceSampler )
+    {
+        super( honeycombConfiguration, defaultTraceSampler );
+    }
+
+    public void addSpanFields ( List<RootSpanFields> fields )
```

Comment:
pls use the registerRootSpanFields in the super class. which is just for cases when CDI is not provided.